README.rst

```diff
@@ -0,0 +1,70 @@
+Image Modal XBlock
+==================
+A fullscreen image modal XBlock.
+
+This package is meant to be installed within the OpenEdX platform.
+
+.. image:: ./docs/lms-view-normal.png
+
+
+Installation, system administrator
+----------------------------------
+
+To install the XBlock on your platform,
+add the following to your `requirements.txt` file:
```

Comment:
Note: I'm intentionally keeping this ambiguous and not going into minutia about `base.in` vs `stanford.txt`, etc..
If you don't know how to install a new package, you're not the target audience here..